examples/kubernetes/dynamic_provisioning/specs/storageclass.yaml

```diff
@@ -0,0 +1,12 @@
+kind: StorageClass
```

Comment:
Do you mind also removing the storageclass and all references to it from the `static_provisioning` example? Now that the driver supports both, having a SC there will only cause confusion. https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/784

Reply:
actually I can do this in a follow up